README.md

```diff
@@ -267,22 +272,22 @@ inspector.validate(schema2, c2); // Invalid: "@.lorem" is missing and not option
 
 ---------------------------------------
 
-### uniqueness
+<h3 id="v_uniqueness">uniqueness</h3>
 
 * **type**: boolean.
 * **default**: false.
 * **usable on**: array, string.
 
 If true, then we ensure no element in candidate exists more than once.
 
-__Example__
+#### Example
```

Comment:
Like above, I like how we prefer markdown syntax and fall back to HTML. Good work.